README.md

```diff
@@ -3,129 +3,123 @@
     <img src="public/images/android-chrome-512x512.png" alt="Logo"  width="150" height="150">
   </a>
  
-  <h1>PairDrop</h1>
+  # _Send it_, with [PairDrop](https://pairdrop.net)
 
   <p>
-    Local file sharing in your browser. Inspired by Apple's AirDrop.
+    Local file sharing <a href="https://pairdrop.net"><strong>in your web browser</strong></a>. 
     <br />
-    <a href="https://pairdrop.net"><strong>Explore  »</strong></a>
     <br />
+    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Report a bug</a>
     <br />
-    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Report Bug</a>
-    ·
-    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Request Feature</a>
+    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Request feature</a>
   </p>
 </div>
 
 ## Features
-[PairDrop](https://pairdrop.net) is a sublime alternative to AirDrop that works on all platforms.
-
-- File Sharing on your local network
-  - Send images, documents or text via peer to peer connection to devices on the same local network.
-- Internet Transfers
-  - Join temporary public rooms to transfer files easily over the internet!
-- Web-Application 
-  - As it is web based, it runs on all devices.
-
-You want to quickly send a file from your phone to your laptop?
-<br>You want to share photos in original quality with friends that use a mixture of Android and iOS?
-<br>You want to share private files peer to peer between Linux systems?
-<br>AirDrop is unreliable again?
-<br>_Send it with PairDrop!_
-
-Developed based on [Snapdrop](https://github.com/RobinLinus/snapdrop)
+File sharing on your local network that works on all platforms.
+
+- A multi-platform AirDrop-like solution that works.
+  - Send images, documents or text via peer-to-peer connection to devices on the same local network.
+- Internet transfers
+  - Join temporary public rooms to transfer files easily over the Internet.
+- Web-app 
+  - Works on all devices with a web browser (that supports HTML5, CSS 3, JS, and WebRTC).
+ 
+Send a file from your phone to your laptop?
+<br>Share photos in original quality with friends using Android and iOS?
+<br>Share private files peer-to-peer between Linux systems?
 
-## Differences to Snapdrop
-<details><summary>Click to expand</summary>
+## Differences to the [Snapdrop](https://github.com/RobinLinus/snapdrop) it is based on
+<details><summary>List view</summary>
 
-### Paired Devices and Public Rooms - Internet Transfer
-* Transfer files over the internet between paired devices or by entering temporary public rooms.
-* Connect to devices in complex network environments (public Wi-Fi, company network, Apple Private Relay, VPN etc.).
+### Paired Devices and Public Rooms — Internet Transfer
+* Transfer files over the Internet between paired devices or by entering temporary public rooms.
+* Connect to devices in complex network environments (public Wi-Fi, company network, Apple private relay, VPN, etc.).
```

Comment:
It's actually neither of ours. It's written "iCloud Private Relay" (see https://support.apple.com/en-us/102602)